buildozer/targets/android.py

```diff
@@ -535,7 +537,7 @@ def _install_p4a(self):
         options = "--user"
         if "VIRTUAL_ENV" in os.environ or "CONDA_PREFIX" in os.environ:
             options = ""
-        cmd('pip install -q {} {}'.format(options, " ".join(pip_deps)))
+        cmd('{} install -q {} {}'.format(PIP_COMMAND, options, " ".join(pip_deps)))
```

Comment:
Or would it make sense to `python -m pip install -q ...` instead? I think that would pick the correct pip module based on the Python version running.
That way we could skip that `PIP_COMMAND = 'pip if...` thing above. What do you think?